shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -555,7 +555,8 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
     File dstFile = new File(dstPath.getPath());
     String randomSuffix =
         String.format(".%s_copyToLocal_", RandomStringUtils.randomAlphanumeric(8));
-    File tmpDst = new File(Name.HOME + randomSuffix);
+    File tmpDst = new File(PathUtils.concatPath( dstFile.getAbsolutePath(),
```

Comment:
@aaudiber  what if the file exists? Should we throw a error or overwrite the file?